src/extensions/default/analyzers/Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers.Common/SyntaxNodeExtensions.Predicates.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.CodeAnalysis;
+using CS = Microsoft.CodeAnalysis.CSharp;
+using CSSyntax = Microsoft.CodeAnalysis.CSharp.Syntax;
+using VB = Microsoft.CodeAnalysis.VisualBasic;
+using VBSyntax = Microsoft.CodeAnalysis.VisualBasic.Syntax;
+
+namespace Microsoft.DotNet.UpgradeAssistant.Extensions.Default
+{
+    public static partial class SyntaxNodeExtensions
+    {
+        public static bool IsCSharp(this SyntaxNode node) => node?.Language == LanguageNames.CSharp;
```

Comment:
Is the `.Predicates` because of the `Is` prefix?

Reply:
yes. Just splitting up the file, as it seemed to be getting large and unwieldy.